pkg/api/vmtemplate/store.go

```diff
@@ -0,0 +1,138 @@
+package vmtemplate
```

Comment:
since this is templateVersionStore so use a meaningful file name instead of `store.go`

Reply:
updated